ciscoise_consts.py

```diff
@@ -54,10 +53,24 @@
 CISCOISE_SUCC_LIST_ENDPOINTS = "{0} Endpoints found"
 CISCOISE_SUCC_GET_ENDPOINT = "Endpoint found"
 CISCOISE_SUCC_UPDATE_ENDPOINT = "Endpoint updated"
+CISCOISE_ERR_INVALID_PARAM = "Please provide non-zero positive integer in {param}"
```

Comment:
Please provide a non-zero positive integer in {param}

Reply:
Updated